Jenkinsfile_nightly

```diff
@@ -24,6 +24,14 @@ withNightlyPipeline(type, product, component) {
   loadVaultSecrets([
     'reform-scan-aat': [
       secret('test-signing-key-der-base64', 'SIGNING_KEY_DER_BASE64')
+    ],
+    'bulk-scan-aat': [
+      secret('idam-users-bulkscan-username', 'IDAM_USERS_BULKSCAN_USERNAME'),
+      secret('idam-users-bulkscan-password', 'IDAM_USERS_BULKSCAN_PASSWORD'),
+      secret('idam-client-secret', 'IDAM_CLIENT_SECRET')
+    ],
+    's2s-aat': [
+      secret('microservicekey-bulk-scan-orchestrator', 'S2S_SECRET')
```

Comment:
we should not use s2s vault. We should already have this value in our vault ?

Reply:
Yes, indeed